db/migrate/20151121215520_create_ratings.rb

```diff
@@ -0,0 +1,11 @@
+class CreateRatings < ActiveRecord::Migration
+  def change
+    create_table :ratings do |t|
+      t.references :rating, index: true
+
+      t.references: :rateable, polymorphic: true, index: true
```

Comment:
I'm not sure what I did here (did the old assignment maybe suggest doing this?), so I tried to re-run ``` rails generate model Rating severity:integer ```, but it says that this old migration conflicts with the rails generate model. It gave the option to ``` --force ``` in the terminal, but I'm not sure what that will do/how to do it